client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -24,63 +24,64 @@ function getHoldMessages( translate ) {
 		},
 		TRANSFER_ALREADY_EXISTS: {
 			title: translate( 'Installation in progress' ),
-			description: translate( 'Another installation is already in progress.' ),
-			supportUrl: 'https://wordpress.com/help'
+			description: translate( 'Please wait for the other installation to complete, then try again.' ),
+			supportUrl: 'https://wordpress.com/help',
 		},
+		// Handled as a banner
 		NO_BUSINESS_PLAN: {
 			title: translate( 'Business plan required' ),
 			description: translate( 'This feature is only allowed on sites with a business plan.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			supportUrl: 'https://support.wordpress.com/',
 		},
 		NO_JETPACK_SITES: {
-			title: translate( 'Jetpack site' ),
-			description: translate( 'This feature is not supported on Jetpack sites.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'Jetpack not supported' ),
+			description: translate( 'Try using a different site.' ),
 		},
 		NO_VIP_SITES: {
-			title: translate( 'VIP site' ),
-			description: translate( 'This feature is not supported on VIP sites.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'VIP site not supported' ),
+			description: translate( 'Try using a different site.' ),
 		},
 		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
+			title: translate( 'Private site not supported' ),
+			description: translate( 'Make your site public to resolve.' ),
+			supportUrl: 'https://support.wordpress.com/settings/privacy-settings/',
 		},
 		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
-			supportUrl: 'https://wordpress.com/help'
+			title: translate( 'Flagged site not supported' ),
+			description: translate( "Contact us to review your site's standing to resolve." ),
+			supportUrl: 'https://support.wordpress.com/suspended-blogs/',
 		},
 		NON_ADMIN_USER: {
 			title: translate( 'Admin access required' ),
 			description: translate( 'Only site administrators are allowed to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/user-roles/'
+			supportUrl: 'https://support.wordpress.com/user-roles/',
 		},
+		// Handled as a banner
 		NOT_USING_CUSTOM_DOMAIN: {
 			title: translate( 'No custom domain' ),
 			description: translate( 'Your site must use a custom domain to use this feature.' ),
```

Comment:
@rralian I'll take a step back here as I've finally cleared what those two holds mean.

`NOT_DOMAIN_OWNER`
You're attempting to AT the site, but the site's primary domain was purchased by another user (and therefore is tied to that other user). In other words: "you're not the owner of the current primary domain of this site".

`NOT_USING_CUSTOM_DOMAIN`
Covers two different cases:
1. You don't own any custom domains (and therefore your site doesn't use a custom domain).
2. You own a custom domain but you don't use it as primary domain for this site.

So, I'd keep the things as they are now.
`NOT_DOMAIN_OWNER` stays a list item, and `NOT_USING_CUSTOM_DOMAIN` stays a banner.

Just, seeing as you could get `NOT_USING_CUSTOM_DOMAIN` even if you already own a domain, I'll replace `/domains/add` with `/domains/manage` so that it doesn't look like we're trying to make the user purchase more domains than the strictly needed one (as you've suggested at first 🙂 ).